test/integration/advanced_metrics_test.go

```diff
@@ -12,12 +12,12 @@ import (
 	"testing"
 	"time"
 
+	conf "github.com/nginx/agent/test/integration/nginx"
+	"github.com/nginx/agent/test/integration/upstream"
+	"github.com/nginx/agent/test/integration/validator"
```

Comment:
I'm not quite sure why the path for these changed from `github.com/nginx/agent/v2` to `github.com/nginx/agent`. At least no tests broke as a result, and everything compiles normally.